opm/material/densead/DynamicEvaluation.hpp

```diff
@@ -518,22 +518,22 @@ class Evaluation<ValueT, DynamicSize>
 
 private:
 
-    FastSmallVector<ValueT, 9> data_;
+    FastSmallVector<ValueT, staticSize> data_;
 };
 
-template <class Scalar>
```

Comment:
please default this parameter to `0`. I guess it would be surprising if the performance suddenly dropped when going to 9 derivatives...

Reply:
okay. You want us to specify this parameter in the downstream?

Reply:
the users of `DynamicEvaluation` should explicitly specify it, yes.